src/test/java/org/fundacionjala/coding/wilson/PlanetTwisted37Test.java

```diff
@@ -0,0 +1,32 @@
+package org.fundacionjala.coding.wilson;
+import java.util.Arrays;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * @author wiltrouble
+ */
+
+public final class PlanetTwisted37Test {
+
+    //private static final Integer[] NUMBERSTEST1 = new Integer[]{1, 2, 7, 4, 5, 6, 3, 8, 9};
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)